### PR TITLE
Refactor highchart defaultOptions

### PR DESCRIPTION
**Pull request by zhb000**
Refactor highchart defaultOptions

## Areas changed

npm packages changed:

- [ ] responsibleai/causality
- [x] responsibleai/core-ui
- [ ] responsibleai/counterfactuals
- [ ] responsibleai/dataset-explorer
- [ ] responsibleai/fairness
- [ ] responsibleai/interpret
- [ ] responsibleai/localization
- [ ] responsibleai/mlchartlib
- [ ] responsibleai/model-assessment

Python packages changed:

- [ ] raiwidgets
- [ ] responsibleai
- [ ] erroranalysis
- [ ] rai_core_flask

## Tests

- [ ] No new tests required.
- [ ] New tests for the added feature are part of this PR.
- [ ] I validated the changes manually.

## Documentation:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: libs/core-ui/src/lib/Highchart/HighchartWrapper.tsx [hunk 1]
```diff
@@ -3,10 +3,10 @@
 
 import { ITheme } from "@fluentui/react";
 import _ from "lodash";
-import { mergeStyles } from "office-ui-fabric-react";
+import { getTheme, mergeStyles } from "office-ui-fabric-react";
```
Comment (imatiach-msft) [inline libs/core-ui/src/lib/Highchart/HighchartWrapper.tsx:6]: why do we need both @fluentui/react and office-ui-fabric-react, I thought they were the same package?  Just for my understanding.

File: libs/core-ui/src/lib/Highchart/getDefaultHighchartOptions.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        style: {
+          color: colorTheme.fontColor
+        }
+      },
+      title: {
+        text: undefined
+      }
+    },
+    yAxis: {
+      gridLineWidth: 1,
```
Comment (imatiach-msft) [inline libs/core-ui/src/lib/Highchart/getDefaultHighchartOptions.ts:80]: the x and y axis look similar and I see this code elsewhere, is there some way to refactor to some common function that returns this info?

**Review by imatiach-msft — APPROVED** [review]

File: libs/core-ui/src/lib/Highchart/HighchartWrapper.tsx (hunk 1 above)
Comment (zhb000) [reply libs/core-ui/src/lib/Highchart/HighchartWrapper.tsx:6]: They are, we are in the process of migrating from fabric ui to fluentui, eventually fluentui will be the one always imported from.

File: libs/core-ui/src/lib/Highchart/getDefaultHighchartOptions.ts (hunk 2 above)
Comment (zhb000) [reply libs/core-ui/src/lib/Highchart/getDefaultHighchartOptions.ts:80]: this one is defaultOptions, each specific chart might have seperate data, labels passed in

**Pull request merged**